.cargo/config.toml

```diff
@@ -35,4 +35,13 @@ linker = "lld-link.exe"
 linker = "lld-link.exe"
 
 [target.aarch64-uwp-windows-msvc]
-linker = "lld-link.exe"
\ No newline at end of file
+linker = "lld-link.exe"
+
+[target.x86_64-unknown-linux-gnu]
+rustflags = ["-C", "link-arg=-fuse-ld=gold"]
+
+[target.i686-unknown-linux-gnu]
+rustflags = ["-C", "link-arg=-fuse-ld=gold"]
```

Comment:
A couple comments about this:
 - This doesn't seem to handle Darwin.
 - I have opened #29912 which perhaps means that these flags can be added platform-independently.

Reply:
There are still android builds ...